introduction.md

```diff
@@ -1,3 +1,8 @@
 I am Jeremy, hear me roar... or not... I tend not to roar, but I do enjoy tea.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?